cmd/node.go

```diff
@@ -110,6 +110,12 @@ func (node *Node) Start() {
 		panic(err)
 	}
 
+	// Make sure there is no prefix conflict in BadgerDb
+	err = lib.CheckForPrefixOverlap()
```

Comment:
Right now this test only works if someone remembers to add their new prefix to the tester function. This isn't robust because people will forget to do this, and the value of the test will be lost.

Instead, what you want is to make it so that a compile error occurs somehow if you forget to add the prefix to the test.

Here's one idea:
```
// Define NEXT_TAG as a variable
NEXT_TAG = 57

// Increment through to NEXT_TAG in the test
for ii := 0; ii < NEXT_TAG; ii++ {
  _, inPrefixMap := prefixMap[ii]
  if !inPrefixMap {
    // Allow for any exceptions
    // return an error if not covered by exceptions
  }
}
```

Reply:
Also nitpick: All the testing code should live in a test file not in the live code.

Reply:
Okay, updated the checks. I feel like it's harder to miss this check if it's in db_utils vs db_utils_test

Reply:
Actually, I thought about this and you're right it's cleaner when it's in db_utils_test